src/main/scala/scalajsGames/games/AstroLander.scala

```diff
@@ -102,13 +109,17 @@ case class AstroLander(bounds: Point, resetGame: () => Unit) extends Game {
     }
 
   }
+
+
   var lastKeys: Set[Int] = Set()
+
   def update(keys: Set[Int]) = {
     lastKeys = keys
     if (fuel > 0) {
-      if (keys(37)) craftVel += Point(0.5, 0).rotate(theta + math.Pi / 4)
-      if (keys(39)) craftVel += Point(0.5, 0).rotate(theta - math.Pi / 4)
-      if (keys(40)) craftVel += Point(0.5, 0).rotate(theta)
+      // I changed the first parameter of Point of 0.5 to 0.1, thats makes the ship slowest
```

Comment:
I'd write this instead `// this handles the ship speed, smaller values lead to slower speeds`